src/setCompliment.js

```diff
@@ -0,0 +1,9 @@
+export default function setCompliment(arr1, arr2){
+  const concatSortArray = arr1.concat(arr2).sort()
+  const results = []
```

Comment:
You could do like a check at the beginning of the function to see if the arguments are array's